src/Import/src/FMI3/fmi3_import.c

```diff
@@ -116,6 +129,21 @@ fmi3_import_t* fmi3_import_parse_xml(
     }
     context->callbacks->free(xmlPath);
 
+   // Only parse terminals and icons of parsing of model description did not fail
+    if (fmu) {
+        // terminalsAndIcons uses modelDescription for error checks
+        if (fmi3_xml_terminals_and_icons_set_model_description(fmu->termIcon, fmu->md)) {
+            fmi3_import_free(fmu);
+            fmu = 0;
```

Comment:
Curious: Does it matter if we set fmu to `0` or `NULL`? I am only asking this out of consistency as it seems we set to `NULL` usually when there are errors.

Reply:
from stddef.h:

```
#ifndef __cplusplus
#define NULL ((void *)0)
#else   /* C++ */
#define NULL 0
#endif  /* C++ */
```

So, no real practical difference, but I think it increases readability. And, yet, we are not very consistent with using it.

Reply:
Please use NULL everywhere it is a pointer. I've tried to change it at all places I've come across it. It makes it very clear that the type is a pointer and not an int.

(I've not looked at other comments/code here, just stumbled on this thread.)